programs/programs/nc/nc_crisis_intervention/calculator.py

```diff
@@ -0,0 +1,51 @@
+from programs.programs.calc import MemberEligibility, ProgramCalculator, Eligibility
+import programs.programs.messages as messages
+
+
+class CrisisIntervention(ProgramCalculator):
+    expenses = ["rent", "mortgage", "heating"]
+    fpl_percent = 1.5
+    percent_of_fpl = 4
+    # resource_limit = 2250
+    large_household_size = 4
+    max_value_fpl_percent = 0.5
+    small_household_low_income_value = 400
+    small_household_large_income_value = 300
+    large_household_low_income_value = 500
+    large_household_large_income_value = 400
+
+    dependencies = [
+        "household_size",
+        "income_amount",
+        "zipcode",
+        "household_size",
+        "age",
```

Comment:
You don't use age or zip code.
```suggestion
        "household_size",
```